src/workerd/util/sqlite.h

```diff
@@ -20,6 +20,15 @@ namespace workerd {
 using kj::byte;
 using kj::uint;
 
+// Used to collect periodic metrics about queries and size of sqlite db
+class SqliteObserver {
+public:
+  virtual void addQueryStats(uint64_t rowsRead, uint64_t rowsWritten) {}
+  virtual void setSqliteStoredBytes(uint64_t sqliteStoredBytes) {}
```

Comment:
It's a little weird that this method is declared here but not actually called by `SqliteDatabase`. In theory a cleaner approach would be to have a separate interface like `SrsDataSizeObserver` declared in the internal codebase.

But I'm fine with leaving it like this as long as we have a comment explaining that this method is here for convenience but not actually called by `SqliteDatabase`.

Reply:
Added a comment